Dockerfile

```diff
@@ -1,9 +1,14 @@
-FROM openjdk:8-jre
+FROM hmcts/cnp-java-base:openjdk-jre-8-alpine-1.4
+
+# Mandatory!
+ENV APP service-auth-provider.jar
+ENV APPLICATION_TOTAL_MEMORY 1024M
+ENV APPLICATION_SIZE_ON_DISK_IN_MB 66
 
 RUN mkdir -p /opt/app
 
-COPY build/install/service-auth-provider /opt/app
+COPY build/libs/$APP /opt/app/
 
 EXPOSE 8489
 
-ENTRYPOINT [ "/opt/app/bin/service-auth-provider" ]
+ENTRYPOINT exec java ${JAVA_OPTS} -jar "/opt/app/service-auth-provider.jar"
```

Comment:
isn't this entrypoint not needed anymore?

Reply:
In the build is failing to find it on the copy command so i guess it has been broken for a while